python/lsst/obs/comCam/comCamMapper.py

```diff
@@ -35,24 +35,53 @@ class ComCamMakeRawVisitInfo(MakeRawVisitInfo):
 
     def setArgDict(self, md, argDict):
         """Fill an argument dict with arguments for makeVisitInfo and pop associated metadata
+
+        Parameters
+        ----------
+        md : `lsst.daf.base.PropertyList or PropertySet`
+            image metadata
+        argDict : `dict`
+            The argument dictionary used to construct the visit info, modified in place
         """
         super(ComCamMakeRawVisitInfo, self).setArgDict(md, argDict)
+        argDict["darkTime"] = self.popFloat(md, "DARKTIME")
+        #
+        # Done setting argDict; check values now that all the header keywords have been consumed
+        #
```

Comment:
Yeah, honestly, that was Robert's comment, and he always does that, I don't know why. Removed.